.github/workflows/check_package.yml

```diff
@@ -0,0 +1,38 @@
+name: Swift-Package
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/**
+    paths:
+      - Package.swift
+
+jobs:
+  build:
+    name: Swift ${{ matrix.swift }}
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        swift: ["5.3", "5.3.1", "5.3.2", "5.4", "5.4.2", "5.5", "5.8"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: swift-actions/setup-swift@v1.23.0
+        with:
+            swift-version: ${{ matrix.swift }} 
+      - name: Build & test MacOS
+        run: |
+          swift build -v
+          swift test -v
```

Comment:
I do not know how this works when trying to build the binary swift package which is generated from swift version 5.3.

Reply:
Umm ... we are doing something similar here but it works.

https://github.com/couchbase/couchbase-lite-swift-ee/blob/master/.github/workflows/swift.yml